db/seeds.rb

```diff
@@ -46,6 +46,13 @@ def random_record_for_org(org, klass)
   partner_key: 'kit'
 )
 
+# ----------------------------------------------------------------------------
+# NDBN Members
+# ----------------------------------------------------------------------------
+NDBNMember.create!(ndbn_member_id: 100, account_name: "Pawnee")
+NDBNMember.create!(ndbn_member_id: 200, account_name: "Other Spot")
```

Comment:
Going with that, I wonder if we should add a validation to the ndbn_member_id to ensure it is 5 digits?

Reply:
I don't think it makes sense, in the case that NDBN decides to break way from 5 digits, I think they should be able to without causing the system to not work.